README.md

```diff
@@ -103,12 +106,12 @@ Lead Developer - Marcelo Vinicius
 
 ## Usage
 
-1. In settings.py add following Middleware bellow:
+1. In your application `settings.py` file, add following Middleware:
 
 ```python
 MIDDLEWARE = [
     #...
-    'django_keycloak_auth.middleware.KeycloakMiddleware',
+    'django-keycloak-auth.middleware.KeycloakMiddleware',
```

Comment:
It seems to me that we need to leave underscores here if we keep a replacement in the module name?

Reply:
That's right. I'll fix it.